lldb/source/Plugins/ExpressionParser/Swift/SwiftExpressionParser.cpp

```diff
@@ -1050,8 +1050,8 @@ MaterializeVariable(SwiftASTManipulatorBase::VariableInfo &variable,
     // this check scattered in several places in the codebase, we should at
     // some point centralize it.
     lldb::StackFrameSP stack_frame_sp = stack_frame_wp.lock();
-    std::optional<uint64_t> size =
-        variable.GetType().GetByteSize(stack_frame_sp.get());
+    auto size = llvm::expectedToOptional(
+        variable.GetType().GetByteSize(stack_frame_sp.get()));
```

Comment:
why is this not a cherry-pick? This must have been necessary on the old stable branch, too?

Reply:
Thanks, I didn’t think to look because this change is very recent and originally targeted `next`.

Reply:
Why was it cherry-picked to the old stable branch?

Reply:
LLVM/clang usually work on tip and then cherry-pick to stable. Though for swift-specific changes it's usually the other way around (since only stable actually builds)

Reply:
The rest made it into `next`:
* https://github.com/swiftlang/llvm-project/pull/10212
* https://github.com/swiftlang/llvm-project/commit/46236f4
* https://github.com/llvm/llvm-project/pull/130091
* https://github.com/swiftlang/llvm-project/commit/2317a72
* https://github.com/swiftlang/llvm-project/commit/98c279a